packages/expo-permissions/ios/EXPermissions.podspec

```
@@ -3,22 +3,87 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name           = 'EXPermissions'
-  s.version        = package['version']
-  s.summary        = package['description']
-  s.description    = package['description']
-  s.license        = package['license']
-  s.author         = package['author']
-  s.homepage       = package['homepage']
-  s.platform       = :ios, '10.0'
-  s.source         = { git: 'https://github.com/expo/expo.git' }
-  s.source_files   = 'EXPermissions/**/*.{h,m}'
-  s.preserve_paths = 'EXPermissions/**/*.{h,m}'
-  s.requires_arc   = true
-
-  s.dependency 'UMCore'
-  s.dependency 'UMPermissionsInterface'
+  s.name            = 'EXPermissions'
+  s.version         = package['version']
+  s.summary         = package['description']
+  s.description     = package['description']
+  s.license         = package['license']
+  s.author          = package['author']
+  s.homepage        = package['homepage']
+  s.platform        = :ios, '10.0'
+  s.source          = { git: 'https://github.com/expo/expo.git' }
 
-end
+  s.requires_arc    = true
+
+  s.default_subspec = "All"
+
+ s.subspec "Core" do |ss|
+    s.dependency 'UMCore'
+    s.dependency 'UMPermissionsInterface'
+    ss.source_files        = "EXPermissions/*.{h,m}"
```

I'm not a ruby expert, but from why these spaces?
```suggestion
    ss.source_files = "EXPermissions/*.{h,m}"
```